mucgpt-frontend/src/components/AppSidebar/AppSidebar.module.css

```
@@ -0,0 +1,353 @@
+    display: flex;
+    align-items: center;
+    justify-content: flex-start !important;
+    width: 100%;
+    text-align: left;
+    color: var(--colorNeutralForeground1) !important;
+}
+
+.secondaryAccordion :global(.fui-AccordionHeader__button) :global(svg),
+.utilitiesSection :global(.fui-AccordionHeader__button) :global(svg) {
+    color: var(--colorNeutralForeground1) !important;
+    fill: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)